### PR TITLE
Move totalSupply, move member defs to top

### PR DESCRIPTION
**Pull request by bogomazov**
Also, need to remove `uint256 public totalSupply;  ` from interface (I am using GitHub editor, so I don't know how to commit multiple file changes online)

### CODE REVIEW

File: contracts/eip20/EIP20.sol [hunk 1]
```diff
@@ -72,7 +76,4 @@ contract EIP20 is EIP20Interface {
     view public returns (uint256 remaining) {
       return allowed[_owner][_spender];
     }
-
```
Comment (maurelian) [inline contracts/eip20/EIP20.sol]: @skmgoldin @simondlr 

I agree, it does seem a bit odd to have these mappings at the bottom. Is there a reason for it?

**Comment by maurelian** [comment]
@bogomazov Sorry for not responding sooner, and thank you for submitting this patch. 

I've opened discussion on the diff, but if we want to move forward with it, we'll ask that you merge this to staging first.

**Comment by skmgoldin** [comment]
@bogomazov can you make this PR to staging? Otherwise we'll close it.

**Comment by maurelian** [comment]
> can you make this PR to staging?

It turns out we can do that from the `edit` button by the title.

**Comment by maurelian** [comment]
I had to roll this back via force push. :(

This fixed master, but staging didn't need the fix. We should try to reduce the time between merges to staging and merging to master.

**Pull request merged**